examples/PythiaConnector.cpp

```diff
@@ -43,7 +43,7 @@ PythiaConnector::PythiaConnector(const char* fname) : m_store(podio::EventStore(
   if (access(fname, F_OK) != -1) {
     m_reader.openFile(fname);
   } else {
-    throw "File not found";
+    PAPASLOG_ERROR("File not found");
```

Comment:
Is PAPASLOG_ERROR doing the throw?
If not are you sure we should remove the throw?

Reply:
The throw was already removed (its in pink with a - sign &  not green with a +) :-)